sycl/CMakeLists.txt

```diff
@@ -13,7 +13,7 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules")
 include(AddSYCLExecutable)
 
 set(SYCL_MAJOR_VERSION 5)
-set(SYCL_MINOR_VERSION 1)
+set(SYCL_MINOR_VERSION 2)
```

Comment:
This isn't a necessary change. @romanovvlad or @s-kanaev will uplift this version for all changes before next release is tagged.

Reply:
Thank you! Fixed in 651c7a49d5faf1eb2d8aba8a314fb220a3fa9179